cl/search/migrations/0013_add_docket_entry_time_filed.sql

```diff
@@ -0,0 +1,128 @@
+BEGIN;
+--
+-- Remove trigger snapshot_insert from model docketentry
+--
+DROP TRIGGER IF EXISTS pgtrigger_snapshot_insert_2de73 ON "search_docketentry";
+--
+-- Remove trigger snapshot_update from model docketentry
+--
+DROP TRIGGER IF EXISTS pgtrigger_snapshot_update_d8176 ON "search_docketentry";
+--
+-- Add field time_filed to docketentry
+--
+ALTER TABLE "search_docketentry" ADD COLUMN "time_filed" time NULL;
+--
+-- Add field time_filed to docketentryevent
+--
+ALTER TABLE "search_docketentryevent" ADD COLUMN "time_filed" time NULL;
```

Comment:
Now that we have the new `event` tables, things have gotten a bit more complex again. We don't keep event tables on most of our replicated servers, but we do keep it on our own replica. 

What that means is that we now need two SQL files and the migration file:
 - The migration file is used on prod.
 - The first SQL file is used on our own replica (it does the stuff we've always done)
 - The second SQL file handles the events and is used on our replica (but not our customer replicas).

So:
 - the migration file is good.
 - this SQL file should get split up into regular migration stuff (add the column) and the trigger stuff (add the column to the event table, tweak the triggers).

Reply:
Go it, I've split the SQL files in two, in one file the regular migration that adds the new column and the other file contains all the trigger stuff, let me know if this is ok.

Because it called my attention that the original SQL file had this order:

```
Drop insert trigger
Drop update trigger

Add new column

Create insert trigger
Create update trigger

```

The files now are:
```
Add new column
```

```
Drop insert trigger
Drop update trigger

Create insert trigger
Create update trigger
```

Does the order the SQL files are applied affect? Like first drop the trigger before adding the new table? and then create the triggers again?

If that's the case, do we need to split the SQL in 3 files? 

```
Drop insert trigger
Drop update trigger
```
```
Add new column
```
```
Create insert trigger
Create update trigger
```

Reply:
Dang, that's a good question. I actually don't know the answer, but I think we can avoid the question by making files for different environments:

So it becomes:
 - One file for prod (the python file, it does regular tables, event tables, and triggers)
 - One file for our replica (an SQL that does regular tables, and event tables, but no triggers since triggers here would risk duplicating events that are already being replicated)
 - One file for replica customers (an SQL file that does just our regular tables)

Reply:
Got it, I've modified the files as suggested:

- `0013_add_docket_entry_time_filed.py` with everything.
- `0013_add_docket_entry_time_filed.sql` the sql migration for our replica, only with the regular and event tables.
- `0013_add_docket_entry_time_filed_customers.sql` the sql migration for customers, only with the regular table.

Let me if that's ok.